recipes/shellcheck/bld.bat

```diff
@@ -0,0 +1,28 @@
+set "BINARY_HOME=%PREFIX%\bin"
+set "PACKAGE_HOME=%PREFIX%\share\%PKG_NAME%-%PKG_VERSION%-%PKG_BUILDNUM%"
```

Comment:
Is this where `stack` packages are being installed? Should we do something like `%PREFIX%\share\stack\...`?

Reply:
Hm, it is not a stack package. It is a Haskel package compiled via GHC... stack just creates the environment to build it. That's why I don't see `stack` as part of directory name. This path would theoretical allow to have several packages in different versions/build numbers in that path. That is anyway not used and at the end of the build file only the resulting binary is kept and the whole `PACKAGE_HOME` folder is deleted before conda could archive it.

Reply:
It might be worth raising an issue over at the `stack` feedstock so that we can determine some best practices for compiling Haskell packages and wrap them up in a `compiler` that everyone can use easily. This is a long term project, but it would really help the Haskell ecosystem in conda-forge to flourish.